examples/ct/handlers.go

```diff
@@ -832,8 +838,8 @@ func parseGetSTHConsistencyRange(r *http.Request) (int64, int64, error) {
 		return 0, 0, err
 	}
 
-	if first <= 0 || second <= 0 {
-		return 0, 0, fmt.Errorf("first and second params cannot be <=0: %d %d", first, second)
+	if first < 0 || second < 0 {
```

Comment:
The spec isn't really that clear, but that's what the existing logs do so it seemed sensible to be consistent.